packages/devtools_app/assets/perfetto/dist/index.html

```diff
@@ -0,0 +1,121 @@
+<!doctype html>
+<html lang="en-us">
+<head>
+  <meta charset="utf-8">
+  <title>Perfetto UI</title>
+  <meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" name="viewport" />
+  <link rel="shortcut icon" id="favicon" type="image/png" href="data:image/png;base64,iVBORw0KGgo=">
+  <link id="devtools-style" rel="stylesheet" href="devtools_dark.css">
+  <script src="devtools_theme_handler.js"></script>
```

Comment:
these two lines are the only custom additions to this index.html file. They are added by the `update_perfetto.sh` script